src/main/java/net/minecraftforge/common/Dimension.java

```diff
@@ -42,7 +42,7 @@ public Dimension(DimensionType type, String dimensionName)
     
     public static void registerDimensions()
     {
-    	registerDimension(-1, new Dimension(DimensionType.NETHER, "minecraft:nether"));
+    	registerDimension(2, new Dimension(DimensionType.NETHER, "minecraft:nether"));
```

Comment:
Changing the id's of vanilla dimensions is probably an extremely bad idea. It will break compatibility with vanilla clients connecting to a forge server with no mods installed.

Reply:
True, but I'm not sure how to register this otherwise. I can shift all the ids up one and return one down I guess. Unless I'm missing something and I can register negative IDs?